src/Lib.hs

```diff
@@ -44,17 +49,78 @@ startPetSelection user goldRemaining = do
 startBattle :: User -> IO ()
 startBattle user = do
     putStrBar
+    putStrLn "YOUR TEAM"
     putStrLn $ show (userRoster user)
-    putStrLn "\nBATTLE"
-    putStrLn "BATTLE"
-    putStrLn "BATTLE"
+    -- TODO add team name?
+
+    -- Create enemy team
+    -- TODO create their lineup using gold the same as the player
+    petOpponent1 <- getPet allPets
+    petOpponent2 <- getPet allPets
+    petOpponent3 <- getPet allPets
+    let opponentRoster = Roster {rosterPet1=(Just petOpponent1), rosterPet2=(Just petOpponent2), rosterPet3=(Just petOpponent3), rosterPet4=Nothing, rosterPet5=Nothing, rosterPet6=Nothing}
```

Comment:
```suggestion
    let 
      opponentRoster = Roster
        { rosterPet1 = Just petOpponent1
        , rosterPet2 = Just petOpponent2
        , rosterPet3 = Just petOpponent3
        , rosterPet4 = Nothing
        , rosterPet5 = Nothing
        , rosterPet6 = Nothing
        }
```

Reply:
another option  here is do define something like this:

```haskell
emptyRoster :: Roster
emptyRoster = Roster Nothing Nothing Nothing Nothing Nothing Nothing
```

then you could simplify this to:

```haskell
    let 
      opponentRoster = emptyRoster
        { rosterPet1 = Just petOpponent1
        , rosterPet2 = Just petOpponent2
        , rosterPet3 = Just petOpponent3
        }
```